server/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java

```diff
@@ -254,6 +254,50 @@ public void testCgroupProbeWithMissingMemory() {
         assertNull(cgroup);
     }
 
+    public void testGetTotalMemFromProcMeminfo() throws Exception {
```

Comment:
Can we also add a test that does not stub out /proc/meminfo and verify that it gives us something > 0? It should run on debian 8 only.